product_packaging_level/models/product_packaging.py

```diff
@@ -126,11 +127,29 @@ def _onchange_package_type(self):
         if package_type_level_id:
             self.packaging_level_id = package_type_level_id
 
-    @api.onchange("packaging_level_id", "package_type_id", "name", "name_policy")
-    def _onchange_name(self):
+    @api.depends(
+        "package_type_id",
+        "name_policy",
+        "packaging_level_id",
+    )
+    @api.depends_context("lang")
```

Comment:
IMO it's a nonsense. You don't want to have this field recomputed every time the user change the language. What I expect is to always have the same value saved into the database whatever the user's language.

Reply:
> My main concern is to ensure to store into the database the value always in the same language..

In fact, if we want to make sure about this, we should enforce the language to be always the default one.
Now, I'm not sure why we want to store this information... is it too expensive to compute?

Reply:
Branch updated, please check again, packaging `name` value that stored in database will based on from `default_language` setted on `product.package.level` so user'slanguage context would not affected them